mobile/lib/models/place_details.dart

```diff
@@ -15,33 +14,24 @@ part 'place_details.g.dart';
 
 @JsonSerializable(explicitToJson: true)
 class PlaceDetails {
-  String name;
-  String location;
+  String name = '';
+  String location = '';
   String siteId;
   String placeId = const Uuid().v4();
   double latitude;
   double longitude;
 
-  PlaceDetails(this.name, this.location, this.siteId, this.placeId,
-      this.latitude, this.longitude);
+  PlaceDetails(
+      {required this.name,
+      required this.location,
+      required this.siteId,
+      required this.placeId,
+      required this.latitude,
+      required this.longitude});
 
   factory PlaceDetails.fromJson(Map<String, dynamic> json) =>
       _$PlaceDetailsFromJson(json);
 
-  String getLocation() {
-    if (location.isNull()) {
-      return '';
-    }
-    return location;
-  }
-
-  String getName() {
-    if (name.isNull()) {
-      return getLocation();
-    }
-    return name;
-  }
-
```

Comment:
Deleted code snippets that generated the location names since they will get gotten from the `location_name`